docs/content/en/docs/tasks/cluster/etcd-backup-restore/bottlerocket-etcd-backup.md

```diff
@@ -0,0 +1,267 @@
+```
+
+### Prepare ETCD nodes for backup and restore
+Install SCP on the ETCD nodes:
+```bash
+echo -n ${ETCD_ENDPOINTS} | xargs -I {} -d" " ssh -o StrictHostKeyChecking=no -i ${SSH_KEY} ${SSH_USERNAME}@{} sudo yum -y install openssh-clients
+```
+
+### Create ETCD Backup
+Make sure to setup the [admin environment variables]({{< relref "#admin-machine-environment-variables-setup" >}}) and [prepare your ETCD nodes for backup]({{< relref "#prepare-etcd-nodes-for-backup-and-restore" >}}) before moving forward.
+
+1. SSH into one of the ETCD nodes
+    ```bash
+    export ETCD_NODE=$(echo -n ${ETCD_ENDPOINTS} | cut -d " " -f1)
+    ssh -i ${SSH_KEY} ${SSH_USERNAME}@${ETCD_NODE}
+    ```
+
+1. Drop into Bottlerocket's root shell
+    ```bash
+    sudo sheltie
+    ```
+
+1. Set these environment variables
+    ```bash
+    # get the container ID corresponding to ETCD pod
+    export ETCD_CONTAINER_ID=$(ctr -n k8s.io c ls | grep -w "etcd-io" | cut -d " " -f1)
+
+    # get the ETCD endpoint
+    export ETCD_ENDPOINT=$(cat /etc/kubernetes/manifests/etcd | grep -wA1 ETCD_ADVERTISE_CLIENT_URLS | tail -1 | grep -oE '[^ ]+$')
+    ```
+
+1. Create the ETCD snapshot
+    ```bash
+    ctr -n k8s.io t exec -t --exec-id etcd ${ETCD_CONTAINER_ID} etcdctl \
+        --endpoints=${ETCD_ENDPOINT} \
+        --cacert=/var/lib/etcd/pki/ca.crt \
+        --cert=/var/lib/etcd/pki/server.crt \
+        --key=/var/lib/etcd/pki/server.key \
+        snapshot save /var/lib/etcd/data/etcd-backup.db
+    ```
+
+1. Move the snapshot to another directory and set proper permissions
+    ```bash
+    mv /var/lib/etcd/data/etcd-backup.db /run/host-containerd/io.containerd.runtime.v2.task/default/admin/rootfs/home/ec2-user/snapshot.db
+    chown 1000 /run/host-containerd/io.containerd.runtime.v2.task/default/admin/rootfs/home/ec2-user/snapshot.db
+    ```
+
+1. Exit out of ETCD node. You will have to type `exit` twice to get back to the admin machine
+    ```bash
+    exit
+    exit
+    ```
+
+1. Copy over the snapshot from the ETCD node
+    ```bash
+    scp -i ${SSH_KEY} ${SSH_USERNAME}@${ETCD_NODE}:/home/ec2-user/snapshot.db ${SNAPSHOT_PATH}
+    ```
+
+You should now have the ETCD snapshot in your current working directory.
+
+### Restore ETCD from Backup
+Make sure to setup the [admin environment variables]({{< relref "#admin-machine-environment-variables-setup" >}}) and [prepare your ETCD nodes for restore]({{< relref "#prepare-etcd-nodes-for-backup-and-restore" >}}) before moving forward.
+
+1. Pause cluster reconciliation
+
+    Before starting the process of restoring ETCD, you have to pause some cluster reconciliation objects so EKS Anywhere doesn't try to perform any operations on the cluster while you restore the ETCD snapshot.
+    ```bash
+    # Pause control plane reconcilation
```

Comment:
can we also include this to existing ubuntu etcd backup doc?

Reply:
yep will add that